core/src/main/java/hudson/cli/ListPluginsCommand.java

```diff
@@ -94,6 +94,6 @@ private void printPlugin(PluginWrapper plugin, int colWidthShortName, int colWid
         }
 
         String formatString = String.format("%%-%ds %%-%ds %%s", colWidthShortName, colWidthDisplayName);
-        stdout.println(String.format(formatString, plugin.getShortName(), plugin.getDisplayName(), version));
+        stdout.printf((formatString) + "%n", plugin.getShortName(), plugin.getDisplayName(), version);
```

Comment:
Are the extra parentheses needed around `formatString`?
```suggestion
        stdout.printf(formatString + "%n", plugin.getShortName(), plugin.getDisplayName(), version);
```